src/patterns/data/default/cyclingCellStates.tsv

```diff
@@ -0,0 +1,21 @@
+defined_class	defined_class_label	cell	pubs	synonym_xrefs	creator	date_created
+CL:4033068	cycling B cell	CL:0000236	PMID:31348891	PMID:31348891	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033069	cycling T cell	CL:0000084	PMID:37291214	PMID:37291214	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033070	cycling dendritic cell	CL:0000451	PMID:32066951	PMID:32066951	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033071	cycling NK cell	CL:0000623	PMID:34062119	PMID:34062119	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033072	cycling gamma-delta T cell	CL:0000798	PMID:32066951	PMID:32066951	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033073	cycling monocyte	CL:0000576	PMID:31348891	PMID:31348891	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033074	"cycling CD8-positive, alpha-beta T cell"	CL:0000625	PMID:34062119	PMID:34062119	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033075	"cycling CD4-positive, alpha-beta T cell"	CL:0000624	PMID:34062119	PMID:34062119	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033076	cycling macrophage	CL:0000235	PMID:33208946	PMID:33208946	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033077	cycling alveolar macrophage	CL:0000583	PMID:37291214	PMID:37291214	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033078	cycling mononuclear phagocyte	CL:0000113	PMID:37468583	PMID:37468583	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033079	cycling endothelial cell of lymphatic vessel	CL:0002138	PMID:37291214	PMID:37291214	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033080	cycling pulmonary alveolar type 2 cell	CL:0002063	PMID:37291214	PMID:37291214	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033081	cycling myeloid cell	CL:0000763	PMID:38301653	PMID:38301653	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4033082	cycling basal cell	CL:0000646	PMID:33208946	PMID:33208946	https://orcid.org/0000-0001-6677-8489	2024-07-02T09:14:23Z
+CL:4047001	cycling stromal cell	CL:0000499	PMID:34497389	PMID:34497389	https://orcid.org/0009-0005-7919-4905	2024-07-19T09:14:23Z
+CL:4047002	cycling glial cell	CL:0000125	PMID:34497389	PMID:34497389	https://orcid.org/0009-0005-7919-4905	2024-07-19T09:14:23Z
+CL:4047003	cycling plasma cell	CL:0000768	PMID:34497389	PMID:34497389	https://orcid.org/0009-0005-7919-4905	2024-07-19T09:14:23Z

```

Comment:
You might want to refresh `definitions.owl`, because it has the label 'cycling immature basophil' for CL_4047003
which is different form the one here in the dosdp filler table.

>Class: <http://purl.obolibrary.org/obo/CL_4047003> (cycling immature basophil)